tests/bulk_update/test_bulk_update_batching.py

```diff
@@ -0,0 +1,35 @@
+import tracemalloc
+
+from django.test import TestCase
+
+from .models import Thing
+
+
+class BulkUpdateMemoryTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        Thing.objects.bulk_create(
+            [Thing(name=f"Thing {i}", description="Old") for i in range(1_000_000)],
```

Comment:
Creating a million objects is probably too slow to run in a test suite or was this just for debugging?